docs/src/content/docs/reference/test.mdx

```diff
@@ -0,0 +1,13 @@
+---
+title: Testing assets
+description: A test
+---
+
+An image with a height of 100px:
+<img src="https://raw.githubusercontent.com/withastro/docs/main/public/default-og-image.png" height="100px" alt="default og " />
+
+An image without a height:
+<img src="https://raw.githubusercontent.com/withastro/docs/main/public/default-og-image.png" alt="default og" />
+
```

Comment:
Can we also try this with a local Markdown image? Astro’s assets supports will add a `height` automatically I think:

```suggestion

A local image, automatically processed by Astro assets support:
![Local image](../../../assets/showcase/felicity.pages.dev.png)

```

Reply:
sure!

Reply:
Thanks! As I suspected this is not looking so lovely 😅 

<img width="470" alt="A rather squished image" src="https://github.com/withastro/starlight/assets/357379/022f88fc-c5b0-4ae7-8224-e5b190c474dd">